src/database.py

```diff
@@ -0,0 +1,179 @@
+import sqlite3
+
+import logging
+
+DB_NAME = "volume/gameguage.db"
+
+
+def create_db(db_name: str = None) -> None:
+    """Create the database if it does not exist."""
+    global DB_NAME
+    if db_name is not None and db_name != DB_NAME:
+        logging.info("Replaced default database name %s with %s", DB_NAME, db_name)
```

Comment:
Python string with `%s` is an old string formatting technique. Use fstring instead.

Reply:
This is not outdated. This is an optimization feature

Reply:
Oh, ofc, I’m sorry for not paying attention!